src/java_tools/junitrunner/java/com/google/testing/coverage/BUILD

```diff
@@ -77,7 +77,7 @@ genrule(
     ],
     outs = ["JacocoCoverage_jarjar_deploy.jar"],
     cmd = "\n".join([
-        "JARJAR=\"$$(mktemp)\"",
+        "JARJAR=\"$$(mktemp 2> /dev/null || mktemp -t bazel)\"",
```

Comment:
@ulfjack we could do that but I'm confused why the `-t` part needs to work on Linux. Presumably the first part, `mktemp 2> /dev/null` without a template will work on Linux, so it should never get to the `mktemp -t bazel` on Linux.